float64 radius
float64 center_x
float64 center_y
float64 z
---
bool success
